docs/06-concepts/06-database/08-transactions.md

```diff
@@ -55,3 +55,62 @@ The available isolation levels are:
 | Serializable | `IsolationLevel.serializable` | Gives the same guarantees as `IsolationLevel.repeatableRead` but also throws if read rows are updated by other transactions. |
 
 For a detailed explanation of the different isolation levels, see the [PostgreSQL documentation](https://www.postgresql.org/docs/current/transaction-iso.html).
+
+## Savepoints
+
+Savepoints allow you to create nested transactions within a transaction. This can be useful when you want to roll back to a specific point in the transaction without rolling back the entire transaction.
+
```

Comment:
Discussion: not sure if this is 100% accurate. Nested transactions is technically another thing. I think the official description can be copied or slightly modified as intro here:

> A savepoint is a special mark inside a transaction that allows all commands that are executed after it was established to be rolled back, restoring the transaction state to what it was at the time of the savepoint.

Reply:
True, that makes more sense.

Reply:
[b4a25f8](https://github.com/serverpod/serverpod_docs/pull/223/commits/b4a25f8de54978eaeb976f05cddaeb2b04d2666b)